.github/workflows/pr_portal-api.yml

```diff
@@ -178,8 +178,8 @@ jobs:
       # https://github.com/cypress-io/github-action/issues/48
       fail-fast: false
       matrix:
-        # run 3 copies of the job in parallel
-        machines: [ 1, 2, 3 ]
+        # run 1 copy of the job
+        machines: [ 1 ]
```

Comment:
in PR, some of these tests are flaky and it looks like this is because we are running the tests in parallel - I've noticed this because locally everything passes.
So I've changed it in GHA as well to run everything on 1 machine